frontend/src/hooks/StudentProfile/useStudentProfileData/useAnimalData.tsx

```diff
@@ -8,6 +8,7 @@ export type NeighboringLevel =
   NeighboringLevelsQuery["getNeighboringLevels"]["currLevel"];
 
 export type AnimalDataResult = {
+  sumOfAllPoints: string | undefined;
```

Comment:
Can you change type?
```suggestion
  sumOfAllPoints: number | undefined;
```